dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3JobTask.java

```diff
@@ -43,8 +48,12 @@ public Ds3Client getDs3Client() {
         return ds3Client;
     }
 
+    public String getSessionName() {
+        return sessionName;
+    }
+
     public boolean isInCache() {
-        return getProgress() == 1;
+        return getProgress() >= 1.0;
```

Comment:
why change from an int to a double?

Reply:
If this is a double, it should use a compare with an error.